.ruby-version

```diff
@@ -1 +1 @@
-2.4.2
+2.6.3
```

Comment:
Would this be a breaking change?

Reply:
This one shouldn't be.  Most of our repos are also on at least this ruby version.